### ISSUE TITLE
iniherit parser read ends with NameError

### ISSUE THREAD

**Issue by Artimi** [cadithealth/iniherit#6]
If I call `iniherit.parser.read()` I get an error 
```
  File "/usr/local/lib/python3.5/site-packages/iniherit/parser.py", line 76, in read
    self._read(fp, filename, encoding=encoding)
  File "/usr/local/lib/python3.5/site-packages/iniherit/parser.py", line 90, in _read
     raw = self._readRecursive(fp, fpname, encoding=encoding)
  File "/usr/local/lib/python3.5/site-packages/iniherit/parser.py", line 146, in _readRecursive
    self._apply(src, ret)
  File "/usr/local/lib/python3.5/site-packages/iniherit/parser.py", line 157, in _apply
    self._im_setraw(dst, self.IM_DEFAULTSECT, option, value)
  File "/usr/local/lib/python3.5/site-packages/iniherit/parser.py", line 175, in _im_setraw
    if six.PY3 and hasattr(dst, '_interpolation'):
NameError: name 'dst' is not defined
```
This is caused by commit 3181a89236c928fe7eb9da78767e5957e84ca5cc  where method `_apply()` is split to two function and second function `_im_setraw()` uses `dst` that is argument of `_apply()` but not `_im_setraw()`. I found this using `yoyo-migrations`. Please fix it asap because without specifying version to `0.3.4` yoyo fails (and this package also).

**Comment by grabner** [comment]
thanks for catching that (we're still on Python 2, so did not notice the compatibility issue with 3). working on it. (eventually, we'll start using travis to catch this sort of thing...)

**Comment by grabner** [comment]
fixed in 0.3.6 -- thanks again for letting us know.

**Comment by Artimi** [comment]
@grabner I thank you for quick response and fix.
